roles/reconcile/tasks/reconcile_pods.yaml

```diff
@@ -89,7 +89,7 @@
       etcd_status: "Unavailable"
       etcd_status_reason: "Etcd is trying to establish quorum"
     k8s_status:
-      api_version: etcd.database.coreos.com/v1beta2
+      api_version: etcd.database.coreos.com/v1beta3
       kind: EtcdCluster
```

Comment:
question: do we want this the same namespace as etcd-operator? coreos doesn't seem to fit here.

Reply:
If we are moving away from the coreos namespace, it would break a lot of things in the operator.

But a good catch. I will add a TODO so we can revisit this topic. WDYT?

Reply:
Created #4 to track the discussion